src/Qir/Tests/QIR-dynamic/CMakeLists.txt

```diff
@@ -1,29 +1,22 @@
-set(TEST_FILES
-  qsharp/obj/qsharp/qir-test-random.bc
-)
+set(TEST_FILES qsharp/obj/qsharp/qir-test-random.bc)
 
 #==============================================================================
 # This executable target links test code against the dynamic libraries rather than the explicit
 # static QIR/RT libs (qir will statically link in the bridge via transitivity of target_link_libraries).
 #
-add_executable(qir-dynamic-tests
-  qir-driver.cpp
-)
+add_executable(qir-dynamic-tests qir-driver.cpp)
 
 foreach(file ${TEST_FILES})
   target_source_from_qir(qir-dynamic-tests ${file})
 endforeach()
 
 target_link_libraries(qir-dynamic-tests PUBLIC
-  "-L${runtime_lib_path}"
-  -lMicrosoft.Quantum.Qir.Runtime
-  -lMicrosoft.Quantum.Qir.QSharp.Foundation
-  -lMicrosoft.Quantum.Qir.QSharp.Core
+  "-L${simulator_lib_path}"
+  -lMicrosoft.Quantum.Simulator.Runtime
```

Comment:
Goodness, how many times does CMake require duplicating this declaration?

Reply:
Lots... every consumer needs to specify it :)